crates/bitwarden/src/tool/generators/username.rs

```diff
@@ -0,0 +1,237 @@
+use crate::{error::Result, util::capitalize_first_letter, wordlist::EFF_LONG_WORD_LIST};
+use rand::{seq::SliceRandom, Rng, RngCore};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "mobile", derive(uniffi::Enum))]
+pub enum AddressType {
```

Comment:
I believe this should be `SubaddressType`? It's a bit confusing when it gets appended to the email. Or mayby `AppendType`.

Reply:
This is used in both the Subaddress and the Catch all variants, so calling it SubaddressType might not be great. Something like AppendType could work yeah